cime_config/namelist_definition_drv.xml

```diff
@@ -2154,7 +2158,77 @@
     <type>integer</type>
     <category>aux_hist</category>
     <group>MED_attributes</group>
-    <desc>Auxiliary mediator wav2med file1 number of time samples per file. By default will be 30</desc>
+    <desc>Number of time samples per file.</desc>
+    <values>
+      <value>30</value>
+    </values>
+  </entry>
+
+   <!-- ======= -->
+
+  <entry id="histaux_wav2med_file2_enabled">
+    <type>logical</type>
+    <category>aux_hist</category>
+    <group>MED_attributes</group>
+    <desc>Auxiliary mediator wav2med average history output every day.</desc>
+    <values>
+      <value>.false.</value>
+    </values>
+  </entry>
+  <entry id="histaux_wav2med_file2_flds">
+    <type>char</type>
+    <category>aux_hist</category>
+    <group>MED_attributes</group>
+    <desc>Auxiliary mediator wav2med average history output every day.</desc>
```

Comment:
This description doesn't seem right, plus it smells of fresh paste, plus there is a better one after the values. Remove?